extensions/amp-accordion/amp-accordion.md

```diff
@@ -267,6 +267,18 @@ it collapses all sections of the accordion. To specify a section, add the
 </button>
 ```
 
+### Actions in Bento Mode
```

Comment:
Are we planning on making this syntax standard across all Bento components? This might be something we need to move into a Bento specific section of the documentation or call out in the Bento guide. cc// @sebastianbenz WDYT?

Reply:
Syntax standard. So ideally we'd do it this way:

/1/ Have a separate doc that explain how to call apis. E.g. 

```
const api = element.getApi();
api.callMethod(...);
```

/2/ Each component would (a) reference this doc on how to call generic API, and (b) list its APIs. E.g.

```
Method: toggle(sectionId?: string)
Args:
 - sectionId: string [optional]
```

Reply:
Sounds good - I've updated the WIP guide to include more information about the syntax and I've added a section for [events in Bento in the extension template](https://github.com/ampproject/amphtml/pull/31944/files#diff-267e4699fc8e75cddd0465a9202998273568854dcde6598b2928e7758abed6bcR120). WDYT of this structure?

Reply:
I like it!  So the idea is as follows?

1. The WIP Guide will include generic information on Events in Bento and Working with the API
2. Each component documentation will have information for Events and Working with the API that is specific to that component.  (not all Bento components fire Events or have API methods)
3. Each Component documentation ideally would include a working code snippet of how to use APIs and handle Events